.gitignore

```diff
@@ -7,7 +7,7 @@
 
 #ignore any files in config/, except those in subdirectories.
 /config/*
-!/config/*/*
+!config/**/*/
```

Comment:
No, don't. I added that because it wasn't properly set up before. Now Git
won't ignore changes done to the example folder, like it should, and will
just ignore changes in the root config folder. That way we can add and
change config files.